aten/src/ATen/native/native_functions.yaml

```diff
@@ -1966,6 +1974,14 @@
 
 - func: var(Tensor self, int[1] dim, bool unbiased=True, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
 
+- func: var_mean(Tensor self, bool unbiased=True) -> (Tensor, Tensor)
+  matches_jit_signature: True
+  variants: function, method
```

Comment:
I think we should only add methods if there is a good reason, let's just make this a function.

Reply:
Fixed